src/csharp/Microsoft.Spark.UnitTest/Sql/GenericRowTests.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.Sql;
+using Xunit;
+
+namespace Microsoft.Spark.UnitTest
+{
+    public class GenericRowTests
+    {        
+        [Fact]
+        public void GenericRowTest()
```

Comment:
Can you just do this in `RowTests.cs`?

Reply:
So I should just delete the GenericRowTests.cs file and move the function to RowTests.cs?